src/xcode/ENA/ENA/Source/Client/HTTP Client/HTTPClient+Configuration.swift

```diff
@@ -88,6 +88,7 @@ extension HTTPClient {
 			)
 		}
 
+		@available(*, deprecated)
```

Comment:
This will be completley removed once the story is done?

Reply:
It gets no longer used, I remived it